neural_lam/weather_dataset.py

```diff
@@ -218,9 +218,10 @@ def __getitem__(self, idx):
         # can roll over to next year, ok because periodicity
 
         # Encode as sin/cos
+        seconds_in_year = 365 * 24 * 3600
```

Comment:
maybe we should add a `# NB: this assumes that all years contain 365 days, may want to change this in the future` or something like that?

Reply:
This whole computation has to be handled better, but I agree that we can just make a note of this for now

Reply:
Since this is only required to calculate forcings, I suggest to handle it directly in a seperate script that provides forcings #34. If you are okay with it I would leave it as is for now.

Reply:
yes, maybe you could just add a comment so that we have a note for later?